internal/pkg/initialize/workload.go

```diff
@@ -223,19 +219,27 @@ func (w *WorkloadInitializer) initService(props *ServiceProps) (string, error) {
 	log.Infoln(color.Help(helpText))
 	log.Infoln()
 
-	err = w.addWlToAppAndSSM(props.WorkloadProps, svcWlType)
+	err = w.addSvcToAppAndSSM(app, props.WorkloadProps)
 	if err != nil {
 		return "", err
 	}
 	return manifestPath, nil
 }
 
-func (w *WorkloadInitializer) addWlToAppAndSSM(props WorkloadProps, wlType string) error {
-	// add workload to application
+func (w *WorkloadInitializer) addSvcToAppAndSSM(app *config.Application, props WorkloadProps) error {
+	return w.addWlToAppAndSSM(app, props, svcWlType)
```

Comment:
nit: it's slightly inconsistent that `addSvcToAppAndSSM` doesn't check for the app in store whereas `addJobToAppAndSSM` does. Can we unify the paths somehow? 
(for ex: move the `w.Store.GetApplication(props.App)` out of `addJobToAppAndSSM` to `initJob`) but I'd be open to better implementations :D